GiniVision/Classes/CameraViewController.swift

```diff
@@ -352,6 +403,7 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> ()
     }
     
     @objc fileprivate func importDocument(_ sender: AnyObject) {
+        toolTipView?.dismiss(withCompletion: nil)
```

Comment:
Not really a fan...
What if there's another action method like this? I will have to also remember that I might have to dismiss the tooltip.
I'm not sure if you can find another way, though...

Reply:
I thought about that but I didn't come up with a better option. The toolTip should only be dismissed if "X" or import buttons are pressed.